apps/condo/domains/notification/schema/MessageBatch.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Generated by `createschema notification.MessageBatch 'messageType:Text; title:Text; message:Text; deepLink?:Text; targets:Json; status:Select:created,processing,failed,done; processingMeta?:Json;'`
+ */
+
+const { isArray, isEmpty } = require('lodash')
+
+const { Text, Select } = require('@keystonejs/fields')
+
+const { Json } = require('@condo/keystone/fields')
+const { GQLListSchema } = require('@condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@condo/keystone/plugins')
+
+const { hasValidJsonStructure } = require('@condo/domains/common/utils/validation.utils')
+const { dvAndSender } = require('@condo/domains/common/schema/plugins/dvAndSender')
+
+const { JSON_EXPECT_ARRAY_ERROR, OPERATION_FORBIDDEN } = require('@condo/domains/common/constants/errors')
+
+const access = require('@condo/domains/notification/access/MessageBatch')
+const {
+    CUSTOM_CONTENT_MESSAGE_TYPE,
+    MESSAGE_BATCH_STATUSES,
+    MESSAGE_BATCH_CREATED_STATUS,
+    MESSAGE_BATCH_TYPE_OPTIONS,
+} = require('@condo/domains/notification/constants/constants')
+
+const { sendMessageBatch } = require('../tasks/sendMessageBatch')
+
+const operationForbiddenValidator = ({ fieldPath, addFieldValidationError, operation }) => {
+    if (operation === 'update') addFieldValidationError(`${OPERATION_FORBIDDEN}] Updating ${fieldPath} is forbidden.`)
+}
+
+const MessageBatch = new GQLListSchema('MessageBatch', {
+    schemaDoc: 'Message batch. Adding record here will cause sending message to all targets (user, phone or email) listed in targets via transport detected based on target type.',
+    fields: {
+        messageType: {
+            schemaDoc: 'Message type to use for sending notification',
+            type: Select,
+            options: MESSAGE_BATCH_TYPE_OPTIONS,
+            defaultValue: CUSTOM_CONTENT_MESSAGE_TYPE,
+            isRequired: false,
+            hooks: {
+                validateInput: operationForbiddenValidator,
+            },
+        },
+
+        title: {
+            schemaDoc: 'Title for a the message to send',
```

Comment:
Maybe also provide some details about format. For example, can it be spanned to multiple lines or not, does it supports interpolation etc.
```suggestion
            schemaDoc: 'Common title for messages to be sent',
```